.github/workflows/canary-deploy.yml

```
@@ -1,5 +1,21 @@
 name: Canary Deployment
 
+permissions:
+  contents: read
+  deployments: write
+
+permissions:
```

actionlint (1.7.12)

[error] key "permissions" is duplicated in "workflow" section. previously defined at line:3,col:1

(syntax-check)